crates/ethereum/evm/Cargo.toml

```diff
@@ -38,3 +38,4 @@ serde_json.workspace = true
 [features]
 default = ["std"]
 std = []
+optimism = []
```

Comment:
this crate is ethereum specific, I want to avoid optimism feature here entirely

could we move the op tests to the op crate?

Reply:
Now should be ok, let me know